website/client/src/components/userMenu/profile.vue

```diff
@@ -10,10 +10,12 @@
           class="btn btn-secondary message-icon"
           @click="sendMessage()"
         >
-          <div
-            class="svg-icon message-icon"
-            v-html="icons.message"
-          ></div>
+          <a :href="'/private-messages?uuid=' + user._id">
```

Comment:
can we use a `router-link` component? It'll be rendered as an `a` tag but meant to work with Vue router links https://router.vuejs.org/api/#router-link-props